index.js

```diff
@@ -282,6 +282,17 @@ function readlinkSync(path) {
   return Memory.readUtf8String(buf, n);
 }
 
+function unlinkSync(path) {
+  const { unlink } = getApi();
+
+  const pathStr = Memory.allocUtf8String(path);
+
+  const result = unlink(pathStr);
+  const n = result.value.valueOf();
```

Comment:
Suppose the `valueOf()` here isn't necessary as the type is `int`, so it's always a `Number`, right?